workers/unity/Packages/com.improbable.gdk.mobile/Editor/MobileLaunchConfig.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using UnityEditor;
+
+namespace Improbable.Gdk.Mobile
+{
+    public class MobileLaunchConfig
+    {
+        public bool shouldConnectLocally;
+        public string RuntimeIp => EditorPrefs.GetString(RuntimeIpEditorPrefKey);
+        public string DevelopmentTeamId => EditorPrefs.GetString(DevelopmentTeamIdEditorPrefKey);
+        
+        internal Dictionary<string, string> availableSimulators = new Dictionary<string, string>();
+        internal Dictionary<string, string> availableDevices = new Dictionary<string, string>();
```

Comment:
Why not make these properties which call the relevant functions as that's the only way they are used currently